extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -144,7 +144,7 @@ export class AmpAdApiHandler {
    * @private
    */
   updateSize_(height, width) {
-    this.baseInstance_.attemptChangeSize(height, width, () => {
+    this.baseInstance_.attemptChangeSize(height, width).then(() => {
```

Comment:
we should be sending `embed-size-dened` on catch here. and also clean up `overflowCallback` by getting rid of the `requestedHeight` and `requestedWidth` params